resolwe/flow/engines/celery.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+from . import BaseManager
+from ..tasks import celery_run
+
+try:
+    import celery
+except ImportError:
+    print('Please install Celery using `pip install celery`', file=sys.stderr)
+
+
+__all__ = ['manager']
+
+
+class Manager(BaseManager):
+
+    def run(self, data_id, script, run_sync=False, verbosity=1):
+        celery_run.async(self.executor, data_id, script, verbosity)
```

Comment:
Looks like you are serializing executors here. This is ok, but I think it would be better to be explicit in what state should be serialized (using `__getstate__` on the executor classes etc.).